spanner/spannertest/integration_test.go

```diff
@@ -1145,6 +1145,13 @@ func TestIntegration_ReadsAndQueries(t *testing.T) {
 				{"Daniel"},
 			},
 		},
+		{
```

Comment:
This could use some additional test cases to cover:
- Combination of aggregated values of different types
- Combination of agg_func(*) and agg_func(col_name) combinations
- Several aggregation expressions in combination with a GROUP BY clause